backend/user/serializers.py

```
@@ -0,0 +1,55 @@
+            email=validated_data['email'],
+            username=validated_data['username'],
+            is_active=False
+        )
+        user.set_password(validated_data['password'])
+
+        try:
+            user.full_clean()
+            user.save()
+        except ValidationError as e:
+            print(e)
+            raise serializers.ValidationError(e)
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)